secure-storage-sample/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="index.js"></script>    
+</head>
+<style>
+  body {
+    color: white;

```

Comment:
We can remove this line; once we're using sp-% everywhere (see sp-textfield comment below), everything is theme aware by default.